lib/ i18n/strings.i18n.json

```diff
@@ -0,0 +1,23 @@
+{
+    "app_title": "Kody testowe poli",
+    "title1": "Lista Kodów Kreskowych",
+    "opis": "Opis",
+    "kod": "Kod kreskowy",
+    "kodplus": "Dodaj kod kreskowy",
+    "typ": "Typ kodu",
+    "error1": "Opis nie może być pusty",
```

Comment:
Opakuj to w strukturę error i daj nazwy opisujące co się dzieje. `emptyDescription`, `emptyCode`, `invalidCode`